src/debug.nit

```diff
@@ -3,6 +3,10 @@ import api
 redef class AppConfig
 	# The default player, if any, that will be logged on a new session
 	var debug_player: String is lazy do return value_or_default("app.debug.player", "")
+
+	init do
+		engine_map["nitc"] = new NitcEngine
```

Comment:
Since we're talking about the engine and not the language, `nitc` is imho more appropriate.

And whilst in the near future I can hardly conceive we'll use another engine, maybe later on we'll use another one.